src/Symfony/Bridge/Twig/Node/FormThemeNode.php

```diff
@@ -28,7 +28,7 @@ public function compile(Compiler $compiler)
     {
         $compiler
             ->addDebugInfo($this)
-            ->write('$this->env->getRuntime(\'Symfony\Bridge\Twig\Form\TwigRenderer\')->setTheme(')
+            ->write('$this->env->getRuntime(\'Symfony\Component\Form\FormRenderer\')->setTheme(')
```

Comment:
Hm, I added the check to the FormThemeNode. But doesn't the same apply to all nodes? E.g. RenderBlockNode and even the FormExtension using ` new TwigFunction('csrf_token', array('Symfony\Component\Form\FormRenderer', 'renderCsrfToken')),` that wouldn't work if the user manually used the TwigRenderer?